storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -159,14 +159,6 @@
     public static final String SCHEDULER_DISPLAY_RESOURCE = "scheduler.display.resource";
 
     /**
-     * Initialization parameters for the group mapping service plugin.
-     * Provides a way for a @link{STORM_GROUP_MAPPING_SERVICE_PROVIDER_PLUGIN}
-     * implementation to access optional settings.
-     */
-    @isType(type = Map.class)
-    public static final String STORM_GROUP_MAPPING_SERVICE_PARAMS = "storm.group.mapping.service.params";
```

Comment:
Was this just not used anywhere?

Reply:
It was not used anywhere until this PR - where `FixedGroupsMapping` uses it.